zanata-rest-client/src/main/java/org/zanata/rest/client/FileResourceClient.java

```diff
@@ -50,14 +55,14 @@
 
     }
 
-    public StringSet acceptedFileTypes() {
-        String types = factory.getClient()
+    public List<DocumentType> acceptedFileTypes() {
+        List<DocumentType> types = factory.getClient()
                 .resource(baseUri)
-                .path(FileResource.SERVICE_PATH)
-                .path("accepted_types")
-                .accept(MediaType.TEXT_PLAIN_TYPE)
-                .get(String.class);
-        return new StringSet(types);
+                        .path(FileResource.SERVICE_PATH
+                                + FileResource.ACCEPTED_TYPE_LIST_RESOURCE)
+            .get(new GenericType<List<DocumentType>>() {
+            });
```

Comment:
- [ ] Something strange happened to indentation here. Line 60, 61 and 63 should all start at the same column.